media/js/mozorg/contact-spaces.js

```diff
@@ -416,6 +418,7 @@
                 }
             } else if (state === 'communities') {
                 initContentId = $('#nav-communities li.current').data('id');
+                mozMap.setMapState();
```

Comment:
Hmmm, it worked for me, although it does take a looong time to load. Will test some more.